tskv/src/schema/schemas.rs

```diff
@@ -223,8 +234,19 @@ impl DBschemas {
                             let schema_get = self
                                 .client
                                 .get_tskv_table_schema(&schema.db, &schema.name)
-                                .map_err(|_| MetaError::Retry)?
-                                .ok_or(MetaError::Retry)?;
+                                .map_err(|e| MetaError::Retry { msg: e.to_string() })?;
+                            let schema_get = match schema_get {
+                                None => self
+                                    .client
+                                    .get_tskv_table_schema_by_meta(&schema.db, &schema.name)
+                                    .await
+                                    .map_err(|e| MetaError::Retry { msg: e.to_string() })?
+                                    .ok_or(MetaError::Retry {
+                                        msg: "Table not found when retry update table schema"
```

Comment:
this code makes me confused， map_err()?.ok_or()?

Reply:
Result<Option<T>>